docs/user_guide.md

```diff
@@ -480,6 +480,17 @@ Check the checkbox at the 1st column to select the specified chart versions:
 * Click the `DOWNLOAD` button to download the chart artifact file. Batch operation is not supported.
 * Click the `UPLOAD` button to upload the new chart version for the current chart
 
+#### Adding labels to/remove labels from chart versions

```

Comment:
We can say `charts` instead of `chart versions`

Reply:
I think charts may make some confusion because the label cannot add to a chart. it can only add to a chart's version currently